spec/svg_spec.rb

```diff
@@ -1,19 +1,21 @@
 require 'rails_helper'
 
 RSpec.describe 'SVG files' do
-  Dir[Rails.root.join('**', '*.svg')].reject { |f| f.include?('node_modules') }.each do |svg_path|
-    relative_path = svg_path.sub(Rails.root.to_s, '')
-
-    next if %w[vendor node_modules].include?(relative_path.split('/')[1])
-
+  files = Dir.glob('{app,public}/**/*.svg', base: Rails.root)
+  files.reject! { |f| f.start_with? 'public/assets/' }
```

Comment:
how come we're rejecting files in this directory?

Reply:
Unless I'm mistaken, they're just compiled copies (duplicates) of `app/assets` files.

Reply:
> Unless I'm mistaken, they're just compiled copies (duplicates) of `app/assets` files.

And: Their presence was causing failures for me locally, likely due to stale assets from an older precompile.